src/input/manager.rs

```diff
@@ -1022,13 +847,11 @@ impl Manager {
             log::trace!("Checking config {:?} for device", config.name);
 
             // Check to see if 'auto_manage' is enabled for this config.
-            let auto_manage = {
-                if let Some(options) = config.options.as_ref() {
-                    options.auto_manage.unwrap_or(false)
-                } else {
-                    false
-                }
-            };
+            let auto_manage = config
+                .options
+                .as_ref()
+                .and_then(|options| Some(options.auto_manage.unwrap_or(false)))
```

Comment:
We should fix this clippy warning:

```suggestion
                .map(|options| options.auto_manage.unwrap_or(false))
```

Reply:
It's done. Cool suggestion.